samples/kvsWebRTCClientViewer.c

```diff
@@ -165,6 +195,26 @@ INT32 main(INT32 argc, CHAR* argv[])
         goto CleanUp;
     }
 
+    PRtcDataChannel pDataChannel = NULL;
+    PRtcPeerConnection pPeerConnection = pSampleStreamingSession->pPeerConnection;
+    SIZE_T datachannelLocalOpenCount = 0;
+
+    // Creating a new datachannel on the peer connection of the existing sample streaming session
+    retStatus = createDataChannel(pPeerConnection, pChannelName, NULL, &pDataChannel);
+    if(retStatus != STATUS_SUCCESS) {
+        printf("[KVS Viewer] createDataChannel(): operation returned status code: 0x%08x \n", retStatus);
+        goto CleanUp;
+    }
+    printf("[KVS Viewer] Creating data channel...completed\n");
+
+    // Setting a callback for when the data channel is open
+    retStatus = dataChannelOnOpen(pDataChannel, (UINT64) &datachannelLocalOpenCount, dataChannelOnOpenCallback);
+    if(retStatus != STATUS_SUCCESS) {
+        printf("[KVS Viewer] dataChannelOnOpen(): operation returned status code: 0x%08x \n", retStatus);
+        goto CleanUp;
+    }
+    printf("[KVS Viewer] Data Channel open now...\n");
+
```

Comment:
We should look into cleaning up data channels at the application exit for which we require `dataChannelClose()` API. We can handle that in a new PR.